src/App.js

```diff
@@ -59,7 +60,11 @@ export default function App() {
         return apiMoves;
     }
 
-    function generatePokemon(response, spritePosition) {
+    function refreshPage() {
```

Comment:
you could probably rename this to something like restartGame or along the lines of that, since that is more of what it is logically doing